resources/js/Pages/SignUp.vue

```diff
@@ -3,6 +3,11 @@
     <Menu />
     <Head title="Sign up" />
     <section>
+        <b-message
+                    type="is-danger"
+                    v-if="form.errors.email">
+                        {{ form.errors.email }}
+        </b-message>
         <div class="box">
             <form action="/users" method="POST" @submit.prevent="create">
```

Comment:
```suggestion
            <form @submit.prevent="create">
```

Unneeded attributes because we use Inertia